qiskit/dagcircuit/dagnode.py

```diff
@@ -164,13 +179,52 @@ def qargs(self):
         """
         Returns list of Qubit, else an empty list.
         """
+        warnings.warn(
```

Comment:
Will this be called from `DAGOpNode.__init__`?

Reply:
This should only get called when a `DAGNode` is instantiated directly, `dag_node = DAGNode(...)` and then `dag_node.qargs` is referenced. When a `DAGOpNode` is instantiated, `dag_opnode = DAGOpNode(...)`, a `DAGNode._qargs` will be created, but any reference to `dag_opnode.qargs` will use the `self.qargs` in `DAGOpNode`, so no deprecation warning.